core/domain/role_services.py

```diff
@@ -110,6 +110,7 @@
     feconf.ROLE_ID_RELEASE_COORDINATOR,
     feconf.ROLE_ID_TOPIC_MANAGER,
     feconf.ROLE_ID_BLOG_ADMIN,
+    feconf.ROLE_ID_BLOG_POST_EDITOR
```

Comment:
This role is added because in AdminRoleHandler schema I have written choices to be only allowed from UPDATABLE_ROLES.
Thus missing this will cuase backend tests to fail for blog_dashboard.py.

Reply:
Looks like this list is used for roles that admin can assign. [The name of the variable is misleading*]

Can we use feconf.ALLOWED_USER_ROLES instead?

Reply:
I found that you are using `UPDATABLE_ROLES` in admin.oy so in that case the addition of `ROLE_ID_BLOG_POST_EDITOR` should not be needed as admins are not allowed to assign blog post editor.

@Rijuta-s Can you please confirm?

Reply:
The role should not be added in the `UPDATABLE_ROLES` list as these are the roles directly available in the dropdown in the **admin page** to update the role. Admin **cannot** update the role to blog post editor. The role was only added to `VIEWABLE_ROLES` list, So that admin can see the users assigned the role of the blog post editor.

Reply:
@Nik-09  does removing this role from the list, make blog_dashboard.py tests fail? I think maybe you mean blog_admin.py test if in case there are failures.I have a separate handler to update role via blog_admin.py which  is mostly independent  from the `UPDATABLE_ROLES` list. Will you please recheck and let me know?

Reply:
> Can we use feconf.ALLOWED_USER_ROLES instead?

Yes it worked.
Thank you @DubeySandeep & @Rijuta-s